stanford_profile.info.yml

```diff
@@ -5,65 +5,124 @@ type: profile
 project: Jumpstart
 core: 8.x
 
-base profile: minimal
-
-# install can be turned off later
-install:
-  # core
-  - drupal:action
+dependencies:
```

Comment:
Are you sure you want all of these to be in dependencies? Anything listed in this section cannot be uninstalled whereas the `install` key allows you to uninstall the module at a later time (through the UI or drush). At some point, I imagine we will be offering the ability to turn on and off some modules.

Reply:
hmm. i'll have to check if changing to `install` will produce the same thing

Reply:
Did you ever find out if install was ok? I think it is going to be vital for us to use the install key as we want to allow for some managing of the modules.

https://www.drupal.org/node/2952947

Reply:
I just checked and i was able to uninstall a couple modules defined in the `dependencies` I dont think the system enforces the dependencies on the profile

Reply:
https://www.drupal.org/node/2952947

> After 8.6.0
> As of #2952888: Allow install profiles to require modules, profiles can define an explicit 'install' array in their info file in addition to the 'dependencies' array. The 'dependencies' array will be treated as a set of hard dependencies, meaning that they cannot be uninstalled because the profile depends on them. 

But it _should_ be enforcing that behavior. My ears are up on this. What are we doing that is allowing for the behaviour?

Reply:
I  should have kept reading

> To maintain backwards compatibility, profiles which only define a 'dependencies' array, but not an 'install' array, will have their list of dependencies treated like an 'install' list. This backwards compatibility layer will be removed in Drupal 9, so profile authors should separate their 'install' list from their actual dependencies as soon as possible.

Reply:
Looks like we want to go with the split today.

Reply:
interesting! good find. so we'll definitely want to split it up.

Reply:
I tried with it as `install` and it worked the same. so i switched it. should be good for future